docs/infra/making-infra-changes.md

```diff
@@ -48,9 +54,21 @@ Look in the script files for more details on usage.
 
 ## Using Terraform CLI directly
 
-Finally, if the wrapper scripts don't meet your needs, you can always run `terraform` directly from the root module directory. You may need to do this if you are running terraform commands other than `terraform plan` and `terraform apply`, such as `terraform import`, `terraform taint`, etc. To do this, you'll need to pass in the appropriate `tfvars` and `tfbackend` files to `terraform init` and `terraform apply`. For example, to make changes to the application's service resources in the dev environment, cd to the `infra/app/service` directory and run:
+Finally, if the wrapper scripts don't meet your needs, you can always run `terraform` directly. You may need to do this if you are running terraform commands other than `terraform plan` and `terraform apply`, such as `terraform import`, `terraform taint`, etc. To do this, you'll need to remember to run `terraform init` with the appropriate `tfbackend` file since the root modules are shared across multiple backends. For example, to make changes to the application's service resources in the dev environment:
+
+```bash
+project-root$ cd infra/app/service
```

Comment:
nit: is this `project-root$` nomenclature common? I find a bare `$` slightly more intuitive for "this is a bash command that should be run from the root of the repo"

Reply:
actually gonna keep the "project-root" convention for now since that's used elsewhere in other docs. if it's confusing let me know.

Reply:
I think this is kind of confusing, but I think we could remedy that by defining our code documentation conventions somewhere early in the documentation (like a README). Similar to the way that technical books, like O'Reilly, have a consistent section of every book that says "here's what the code in this book looks like".

Reply:
open to that as a backup option but first choice is something that's intuitive. any suggestions? so far the suggestions include:

- `project-root$ command` - current. some find it confusing.
- `$ command` - leaves out information of what the current path should be.
- `/path/to/project/root$ command` - suggested in earlier comment
- `<project root>$`

FWIW when i ask chat gpt: "In a tutorial that includes shell commands, what is a common convention for representing the path/directory that the reader should run the commands in?" i get the following responses:

<img width="731" alt="image" src="https://github.com/navapbc/template-infra/assets/447859/f135fbfd-a96d-4f58-b556-971a0a2a6dc0">